example/http/server/server.go

```diff
@@ -62,6 +62,7 @@ func main() {
 			trace.WithAttributes(attrs...),
```

Comment:
If we need to set attributes from entries, it would be also good to place here.
`trace.WithAttributes(entries...),`